.gitlab-ci-darwin.yml

```diff
@@ -1,26 +1,49 @@
 variables:
-  SCHEDULER_PARAMETERS: '--nodes=1 --partition=power9 --export=NONE'
+  SCHEDULER_PARAMETERS: '--nodes=1 --partition=power9-eap-ci --export=NONE'
   GIT_SUBMODULE_STRATEGY: recursive
 
 stages:
+  - performance-target
   - performance-regression
 
 .gcc-mpi-cuda-performance-regression:
   variables: 
     CMAKE_CXX_COMPILER: $CI_PROJECT_DIR/external/Kokkos/bin/nvcc_wrapper
   script:
-    - env -i bash --norc --noprofile ./scripts/darwin/build_fast.sh
+    - env
+    - git fetch --all
+    - printf 'env -i bash --norc --noprofile %s/scripts/darwin/build_fast_target.sh  "%s" "%s" "%s" "%s"\n' ${CI_PROJECT_DIR} ${GITHUB_APP_PEM} ${CI_COMMIT_SHA} ${CI_COMMIT_BRANCH} ${CI_JOB_URL}> execute_main.sh
+    - printf 'env -i bash --norc --noprofile %s/scripts/darwin/metrics.sh "%s" "%s/build" "%s" "%s"\n' ${CI_PROJECT_DIR} ${GITHUB_APP_PEM} ${CI_PROJECT_DIR} ${CI_COMMIT_SHA} ${CI_COMMIT_BRANCH}>> execute_main.sh
```

Comment:
> What's the purpose of `execute_main.sh`? Does it actually get executed by something or is it just a log file? Do these two lines exactly duplicate the two commands below? If so, we should only have them once so that they don't need to be kept in sync.
> 
> If `execute_main.sh` is just a log file, maybe a combination of `set -x` and `exec > execute_main.sh` can be used in the commands below to record what's being done. Or maybe we can just execute `execute_main.sh` below instead of providing the full commands again. If you want to stop execution on the first error, I think `set -e` will do that for you.

That's a really good point, the execute main was used to help with debugging the ci, it is a lot easier to do it this way because it save the state of the ci variables, otherwise I have to go look them up, but ya I could simply execute main so I'm not duplicating, great idea.

Reply:
Done.